bindings/py/cpp_src/bindings/encoders/py_RDSE.cpp

```diff
@@ -108,7 +108,9 @@ fields are filled in automatically.)");
         py_RDSE.def_property_readonly("size",
             [](RDSE &self) { return self.size; });
 
-        py_RDSE.def("encode", &RDSE::encode, R"()");
+        py_RDSE.def("encode", [](RDSE &self, htm::Real64 value, htm::SDR* sdr) {
```

Comment:
Actually, the NetworkAPI cannot use  `output = encode(input)` if it is going to avoid another copy because it must create the output buffers at configuration time.

Reply:
Yes, we need both forms.  One is a convenience, the other is needed for in-place assignment.